wrappers/wine-msvc.sh

```diff
@@ -4,7 +4,7 @@ shift
 ARGS=()
 while [ $# -gt 0 ]; do
 	a=$1
-	case $a in
+	case "$a" in
```

Comment:
Hmm, I guess this particular case here is something that is done just in case, but I don't see where a path with spaces would be passed as argument here?

Reply:
Yes, at this point I was just being consistent. Better to have unnecessary quotes than forgetting them somewhere.